SevenRedLibrary/Card.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace SevenRedLibrary
+{
+    /// <summary>
+    /// Class contains object type card
+    /// </summary>
+    public class Card : IComparable<Card>
+    {
+        public int Nominal { get; }
+        public Colors Color { get; }
+        private string[] Combination { get; }
```

Comment:
You should parse input card data from string and save parsed values to `Nomital` and `Color` fields. You don't need to store other magic fields